static/app/views/alerts/rules/metric/details/metricChartOption.tsx

```diff
@@ -355,6 +403,72 @@ export function getMetricAlertChartOption({
       });
   }
 
+  if (anomalies) {
+    const anomalyBlocks: any[] = [];
+    let start;
+    let end;
```

Comment:
start and end could use some types
```suggestion
    let start: string | undefined
    let end: string | undefined
```